cli/commands/archive/archive.go

```diff
@@ -507,10 +530,25 @@ func archiveDownloadVideos(downloader crunchyroll.Downloader, filename string, v
 	return files, nil
 }
 
+func stringInSlice(a string, list []string) bool {
+	for _, b := range list {
+		if b == a {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Will put it in a file in utils - I'm not a fan of on place

Which file name could probably fit best ?
1. helper.go 
2. global.go  
3. query.go (because it a type of searching ) 
4. functions.go

Reply:
Maybe `std.go` - synonym for utils for the standard library

Reply:
Frage wie kann man bei GO denn import von github.com/crunchy-labs/crunchy-cli auf ein fork setzen ohne einen replace über alle Dateien zu machen ?

Reply:
https://go.dev/doc/modules/gomod-ref#replace